tests/ranger/container/test_bookmarks.py

```diff
@@ -56,3 +56,20 @@ def crash():
         secondstore.update_if_outdated()
     secondstore.update = origupdate
     secondstore.update_if_outdated()
+
+
+def test_bookmark_symlink(tmpdir):
+    # Initialize plain file and symlink paths
+    bookmarkfile_link = tmpdir.join("bookmarkfile")
+    bookmarkfile_orig = tmpdir.join("bookmarkfile.org")
```

Comment:
`.org` is a potentially misleading choice as it's used extensively by Emacs org-mode. I'd rather use `.orig`.

Reply:
I agree, I'll make the change.